### DISCUSSION TITLE
Sharing scalalib.publish.License

### DISCUSSION THREAD

**Discussion by jodersky** [com-lihaoyi/mill#4041] [Development]
I'm working on python support, and I think we could reuse the SPDX license abstractions defined in `scalalib.publish.License`. Any suggestion on where the best place would be to extract this?

**Comment by lihaoyi** [comment]
I'd say just copy-paste it for now. We can consider moving it into `mill.main` or something if necessary, but that would be a breaking change and would need to wait until later anyway

**Reply by lefou** [reply]
I'd prefer to just add the dependency to `scalalib` instead of copy-pasting. Most likely, these classes will be loaded in the same classloader anyways. We can introduce a new shared module when we refactor larger parts for `0.13`.

**Reply by lihaoyi** [reply]
Oh yeah that's true, we can just depend on it since its all bundled anyway

**Reply by jodersky** [reply]
Sounds good, that's what I've done in #4032
